### PR TITLE
fix(gorgone): package last version of libzmq

### PR DESCRIPTION
**Pull request by sdepassio**
## Description

We need a bug fix in the latest version of libzmq (4.3.5), but the repositories of the linux distributions we support provides only the 4.3.4.
So we need to package the v4.3.5.

### CODE REVIEW

File: .github/workflows/zmq-lib.yml [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+        run: |
+          dnf install -y wget rpmdevtools rpmlint epel-release
+          dnf config-manager --set-enabled crb || true # alma 9
+          dnf config-manager --set-enabled powertools || true # alma 8
+          dnf install -y asciidoc autoconf automake gcc gcc-c++ glib2-devel libbsd-devel libtool make xmlto
+          cd /github/home
+          wget -O - https://github.com/zeromq/libzmq/archive/refs/tags/v4.3.5.tar.gz|tar zxvf -
+          mkdir -p /github/home/rpmbuild/{BUILD,BUILDROOT,RPMS,SOURCES,SPECS,SRPMS}
+          cp libzmq-4.3.5/packaging/redhat/zeromq.spec /github/home/rpmbuild/SPECS/
+          wget https://github.com/zeromq/libzmq/releases/download/v4.3.5/zeromq-4.3.5.tar.gz -O  /github/home/rpmbuild/SOURCES/zeromq-4.3.5.tar.gz
```
Comment (kduret) [inline .github/workflows/zmq-lib.yml:60]: why do you get git tag and then download release ?

**Review by kduret — COMMENTED** [review]
since gorgone will be moved to centreon-collect repository, can you reopen this PR on centreon-collect repo

File: .github/workflows/zmq-lib.yml [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+          cache_key: ${{ github.sha }}-${{ github.run_id }}-deb-${{ matrix.distrib }}
+          stability: ${{ needs.get-version.outputs.stability }}
+
+  promote:
+    needs: [get-version]
+    if: ${{ contains(fromJson('["stable"]'), needs.get-version.outputs.stability) && github.event_name != 'workflow_dispatch' }}
+    runs-on: [self-hosted, common]
+    strategy:
+      matrix:
+        distrib: [el8, el9, bullseye, bookworm, jammy]
```
Comment (paul-oureib) [inline .github/workflows/zmq-lib.yml:164]: do jammy packages have to be promoted?

**Comment by sdepassio** [comment]
PR moved to centreon-collect

**Pull request closed without merging**